### DISCUSSION TITLE
Audio player

### DISCUSSION THREAD

**Discussion by thias42** [h2oai/wave#757] [Q&A] [answered]
Is there a way to embed and control an audio player on a page? I am currently using a ui.template with an HTML audio element but would like to use more sophisticated audio player like wavesurfer.js and being able to control the source and state of the player dynamically. Is there a way to do this already?

**Comment by lo5** [comment, marked as answer]
Interesting! This should be possible after #297 is implemented. 
wavesurfer.js would make a good example for our docs as well - thanks for the pointer.
